checks/container.go

```diff
@@ -114,11 +112,17 @@ func fmtContainers(
 		sys2, sys1 := ctr.CPU.SystemUsage, lastCtr.CPU.SystemUsage
 
 		// Retrieves metadata tags
-		tags, err := tagger.Tag(docker.ContainerIDToEntityName(ctr.ID), true)
+		entityID := docker.ContainerIDToEntityName(ctr.ID)
+		tags, err := tagger.Tag(entityID, true)
 		if err != nil {
+			log.Infof("Error retrieving tags for container: %s", err)
 			tags = []string{}
 		}
 
+		if _, isPresent := containerServiceTags[entityID]; isPresent {
```

Comment:
nit: go style is usually `_, ok := ...`
also you can do, `serviceTags, ok := ...` and then append `serviceTags...`